ts/private/ts_validate_options.bzl

```diff
@@ -23,6 +24,20 @@ def _validate_options_impl(ctx):
     # We make it a .d.ts file so we can plumb it to the deps of the ts_project compile.
     marker = ctx.actions.declare_file("%s.optionsvalid.d.ts" % ctx.label.name)
 
+    # Provider validation
+    if not ctx.attr.allow_js:
+        for d in ctx.attr.deps:
+            if not d[JsInfo].declarations:
+                fail("""\
+ts_project '{1}' dependency '{0}' does does not contain any declarations (.d.ts or other type-check files).
+Generally, targets which produce no declarations aren't useful as dependencies to the TypeScript type-checker.
+This likely means you forgot to set 'declaration = true' in the compilerOptions for that target.
+
+To disable this check, set the validate attribute to False:
+  npx @bazel/buildozer 'set validate False' {1}
+""".format(d.label, ctx.attr.target))
```

Comment:
could you just post a comment of what this error looks like now when a user hits it, and test that the buildozer command works to repair if you remove the validate = False from that example?

Reply:
```
Error in fail: ts_project '//examples/path_mappings:path_mappings' dependency '//examples/js_lib:lib' does does not contain any declarations (.d.ts or other type-check files).
Generally, targets which produce no declarations aren't useful as dependencies to the TypeScript type-checker.
This likely means you forgot to set 'declaration = true' in the compilerOptions for that target.

To disable this check, set the validate attribute to False:
  npx @bazel/buildozer 'set validate False' //examples/path_mappings:path_mappings
```

Note originally this was `d` instead of `d.label` so the message had `<target //examples/path_mappings:path_mappings>`

Reply:
nice